### ISSUE TITLE
config.static_cache_control deprecation warning

### ISSUE THREAD

**Issue by ijpiantanida** [romanbsd/heroku-deflater#26]
Rails: 5.0.0.1

```
DEPRECATION WARNING: `config.static_cache_control` is deprecated and will be removed in Rails 5.1.
Please use
`config.public_file_server.headers = { 'Cache-Control' => 'public, max-age=604800' }`
instead.
```

**Comment by gagoit** [comment]
Plz use this gem from github:
gem 'heroku-deflater', :group => :production, git: "https://github.com/romanbsd/heroku-deflater.git"
instead of 
gem 'heroku-deflater', :group => :production

**Comment by ijpiantanida** [comment]
That worked, thanks!

Any reason why not releasing a new version?

**Comment by datpb-evolable** [comment]
I have no idea with it :)

**Comment by gagoit** [comment]
I have no idea with it :)

**Comment by bzf** [comment]
Any plans on releasing a new version? :)

**Comment by romanbsd** [comment]
Yep :)

**Comment by vincentwoo** [comment]
Hi @romanbsd, any chance you can cut a release just with this fix? Rails 5 is stable now, so it's probably going to see a lot more usage and this gem is very useful on Heroku.

**Comment by romanbsd** [comment]
I released a new version, 5.0.1 should be supported.
